notebook2/notebook/templates/header.html

```diff
@@ -7,6 +7,20 @@
     Notebook
   </a>
 
+  <a href="{{ batch_base_url }}" class="home link-button">
```

Comment:
Should these all have the class tag "home"?

Reply:
The spacing definitely looks batter with the "home" tag, although I don't actually know what it means semantically.  @akotlar?

Reply:
.home.link-button is a subclass of .link-button that adds extra padding to account for the fact that the home button is a forward slash.

Reply:
Typically we also want to give some special emphasis to the home button / product logo.

Reply:
> .home.link-button is a subclass of .link-button that adds extra padding to account for the fact that the home button is a forward slash.

I'm not sure I understand why a forward slash needs more padding.  Or it adds extra padding to other buttons because the home button is a slash?

 > Typically we also want to give some special emphasis to the home button / product logo.

Does this mean it shouldn't be used on other buttons?  You were also using it on e.g. notebook.

I guess I still don't understand, sorry.

Reply:
> I'm not sure I understand why a forward slash needs more padding. 

Padding is a margin that is included in the definition of the element's dimensions. So by adding a padding you can create a uniform clickable area, regardless of the size of the text inside the element.

> Or it adds extra padding to other buttons because the home button is a slash?

The home button typically has some visual features to distinguish it from other links, so I started with a subclass/modifying style called home. During development I ended up preferring a uniform appearance, and applied the "home" to all. 

> Does this mean it shouldn't be used on other buttons? You were also using it on e.g. notebook.

No, we can use it anywhere we want a link that is uniform in appearance with the header. I would prefer this, our visual language should be as simple as possible to avoid distracting from the core product.